### DISCUSSION TITLE
MHIG_EU_VW_S1538

### DISCUSSION THREAD

**Discussion by VWBOY** [Mr-MIBonk/M.I.B._More-Incredible-Bash#319] [General]
Hi to all friends

How are you?
Is any chance to do CP OFF in the radio sw?
It is Harman
It is my first post in this forum
Nice forum,

**Comment by harman-f** [comment]
Update to latest FW and patch that.

**Reply by VWBOY** [reply]
Hi my dear friend
Pls can you show me a link where to dwld latest fw ?
And pls show me the pach link
Thank you very much indeed

**Comment by harman-f** [comment]
Mibsolution.one

**Comment by VWBOY** [comment]
Hi dear friend

I am searching through your link and can not find for MHIG

Thank you very much

**Comment by VWBOY** [comment]
HI DEAR friend


DO YOU mean this link?
https://mibsolution.one/#/1/9/_pre%20MIB2/MHIG%20-%20HARMAN/Solution

thank you

**Comment by VWBOY** [comment]
Any help for this will be much appreciated

**Reply by VWBOY** [reply]
Hi
I tried to updt with latest versiin
MHIG_EU_VW_K1550 MU0407
It says sw conflict error 
How to solve this pls
